coalib/results/Diff.py

```diff
@@ -351,14 +351,17 @@ def __bool__(self):
     def delete_line(self, line_nr):
         """
         Mark the given line nr as deleted. The first line is line number 1.
+        Raises an error if line nr doesn't exist in the diff.
         """
+        line_nr = 0 if (len(self._file) < line_nr) else line_nr
```

Comment:
line nr 0 or less  than that is considered as invalid line number. Well for a diff of 5 lines, line nr 10 is also invalid line nr.